sdk/lib/crt/setjmp/amd64/setjmp.s

```diff
@@ -1,13 +1,11 @@
 /*
- * COPYRIGHT:         See COPYING in the top level directory
- * PROJECT:           ReactOS system libraries
- * PURPOSE:           Implementation of _setjmp/longjmp
- * FILE:              lib/sdk/crt/setjmp/amd64/setjmp.s
- * PROGRAMMER:        Timo Kreuzer (timo.kreuzer@reactos.org)
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Implementation of setjmp/longjmp
+ * COPYRIGHT:   Copyright (c) Timo Kreuzer (timo.kreuzer@reactos.org)
+ *              Copyright (c) 2025 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
  */
 
-/* INCLUDES ******************************************************************/
```

Comment:
Hum, so we get amd64 with 1 comment line, i386 with 0 and arm with 2...
As well revert this change, or apply it consistently.

Reply:
Restored.